torch/_ops.py

```diff
@@ -83,16 +79,16 @@ def __call__(self, *args, **kwargs):
         return self.dispatch(dispatch_key_set.highestPriorityTypeId(), *args, **kwargs)
 
     def name(self):
-        return name
+        return self.name
 
     # TODO(voz): Should rewrite fallthrough register as the impl for keys we do not specify
     # as opposed to being this sort of explicit thing where ops are a little too key aware...
     def _fallthrough_fn(self, operator, dispatch_key):
         def inner(*args, **kwargs):
-            all_keys_after_current = torch._C._dispatch_keyset_full_after(dispatch_key)
+            all_keys_after_current = torch._C._dispatch_keyset_full_after(dispatch_key)  # type: ignore[attr-defined]
             all_keys_after_current_masked = all_keys_after_current & _compute_keyset(
                 args, kwargs
-            )
+            )  # type: ignore[attr-defined]
```

Comment:
Please don't do this; instead, add the signatures to `torch/_C/__init__.pyi.in`

Reply:
Yeah, that feels better, didn't know how to fix it.